build.gradle

```diff
@@ -285,6 +286,7 @@ project(':iceberg-aws') {
     testCompile("com.adobe.testing:s3mock-junit4") {
       exclude module: "spring-boot-starter-logging"
       exclude module: "logback-classic"
+      exclude group: 'junit'
```

Comment:
Should this be global? It seems like we exclude it quite a bit and could easily have it creep back into the classpath.

Reply:
that doesn't work unfortunately as `junit-vintage-engine` pulls in `junit` itself